### ISSUE TITLE
SwiftUI `Text.init(_ date: NaiveDate, style: Text.DateStyle)` utility

### ISSUE THREAD

**Issue by robmaceachern** [kean/NaiveDate#1]
Thanks for the helpful library. I discovered it as part of Create API and it's wonderful!

I was wondering if you'd be interested in adding a SwiftUI `Text.init(_ date: NaiveDate, style: Text.DateStyle)` utility, in the same style as [Text.init(_ date: Date, style: Text.DateStyle)](https://developer.apple.com/documentation/swiftui/text/init(_:style:)).

I think it'd be helpful (I was reaching for it today) but I'm not sure if it'd be a fit as part of this package.

**Comment by kean** [comment]
Yes, `NaiveDate` has some convenience localizable strings formatting. I don't mind adding it. PRs are welcome.

**Comment by lvalenta** [comment]
I implemented it here: https://github.com/CreateAPI/NaiveDate/pull/6 – however, we can only make `Text.init(_ date: NaiveDate, format: NaiveDate.FormatStyle)`.
